visualization/include/pcl/visualization/pcl_visualizer.h

```diff
@@ -2097,17 +2145,29 @@ namespace pcl
           * \param[in] actor a pointer to the vtk actor object
           * \param[in] viewport the view port where the actor should be removed from (default: all)
           */
+		    PCL_DEPRECATED(1, 14, "deprecated function. Use removeCustomActorFromRenderer instead");
```

Comment:
No `;` at the end

Reply:
And remove "deprecated function"

Reply:
Ahh okay. Thats what the error from the MacOS build meant. Thanks will change it.